spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/writer/WriteConf.scala

```diff
@@ -122,6 +122,12 @@ object WriteConf {
       | per single core in MB/s. <br> Limit this on long (+8 hour) runs to 70% of your max throughput
       | as seen on a smaller job for stability""".stripMargin)
 
+  val TtlParam = ConfigParameter[Int] (
+    name = "spark.cassandra.output.ttl",
+    section = ReferenceSection,
+    default = 0,
+    description = """Time to live in seconds for the columns per insertion""".stripMargin)
```

Comment:
Description may want to say, """Time To Live(TTL) assigned to writes to Cassandra"""